### PR TITLE
Change qpu complex wording

### PR DESCRIPTION
**Pull request by beckykd**
### Summary

Wording changes suggested by @jyu00 and @abbycross 

### Details and comments
Fixes #

### CODE REVIEW

File: docs/faqs/max_execution_time.rst [hunk 1]
```diff
@@ -86,6 +86,4 @@ Other limitations
 
 - Programs cannot exceed 750KB in size.
 - Inputs to jobs cannot exceed 64MB in size.
-- Open plan users are limited to 10 minutes of job execution time per month.  This is the time that the QPU
-   complex (including control software, control electronics, QPU, and so on) is engaged in
-   processing the job. Open plan users can track current progress toward the limit on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
\ No newline at end of file
+- Open plan users get 10 minutes of job execution time per month (reset on the first day of the month).  Job execution time is the amount of time that the system is dedicated to processing your job. You can track progress toward the limit on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
```
Comment (beckykd) [inline docs/faqs/max_execution_time.rst:89]:
```suggestion
- Open plan users get 10 minutes of job execution time per month (reset on the first day of the month).  Job execution time is the amount of time that the system is dedicated to processing your job. You can track your monthly usage on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
```

File: qiskit_ibm_runtime/options/options.py [hunk 2]
```diff
@@ -62,10 +62,10 @@ class Options:
             for more information about the error mitigation methods used at each level.
 
         max_execution_time: Maximum execution time in seconds, which is based
-            on quantum time (not wall clock time). Quantum time is the time that
-            the QPU complex (including control software, control electronics, QPU, and so on)
-            is engaged in processing the job. If a job exceeds this time limit, it is forcibly cancelled.
-            Simulator jobs continue to use wall clock time because they do not have quantum time.
+            on job execution time (not wall clock time). Job execution time is the time that
```
Comment (beckykd) [inline qiskit_ibm_runtime/options/options.py:65]:
```suggestion
            on job execution time (not wall clock time). Job execution time is 
```

File: qiskit_ibm_runtime/runtime_job.py [hunk 3]
```diff
@@ -666,7 +666,7 @@ def usage_estimation(self) -> Dict[str, Any]:
         Returns:
             ``quantum_seconds`` which is the estimated quantum time
```
Comment (beckykd) [inline qiskit_ibm_runtime/runtime_job.py:667]: References to quantum seconds need to be changes as well, but @jyu00 is opening a second PR for this.

Comment (beckykd) [inline qiskit_ibm_runtime/runtime_job.py:667]:
```suggestion
            ``quantum_seconds`` which is the estimated job execution time
```

File: releasenotes/notes/0.11/job-cost-estimation-d0ba83dbc95c3f67.yaml [hunk 4]
```diff
@@ -3,4 +3,4 @@ features:
   - |
     Added a new property, :meth:`~qiskit_ibm_runtime.RuntimeJob.usage_estimation`
     that returns the estimated running time, ``quantum_seconds``. Quantum time
```
Comment (beckykd) [inline releasenotes/notes/0.11/job-cost-estimation-d0ba83dbc95c3f67.yaml:5]:
```suggestion
    that returns the estimated job execution time, ``quantum_seconds``. Job execution time 
```

File: docs/faqs/max_execution_time.rst [hunk 5]
```diff
@@ -86,6 +86,4 @@ Other limitations
 
 - Programs cannot exceed 750KB in size.
 - Inputs to jobs cannot exceed 64MB in size.
-- Open plan users are limited to 10 minutes of job execution time per month.  This is the time that the QPU
-   complex (including control software, control electronics, QPU, and so on) is engaged in
-   processing the job. Open plan users can track current progress toward the limit on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
\ No newline at end of file
+- Open plan users get 10 minutes of job execution time per month (reset on the first day of the month).  Job execution time is the amount of time that the system is dedicated to processing your job. You can track your monthly usage on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
```
Comment (nonhermitian) [inline docs/faqs/max_execution_time.rst:89]: This should have wording that indicates the 10min is a max, e.g "users get at most 10 minutes" or the like.

Comment (beckykd) [reply docs/faqs/max_execution_time.rst:89]: Yeah; we're supposed to document it without using the word "limit", but maybe "up to" or "at most" will be OK

Comment (nonhermitian) [reply docs/faqs/max_execution_time.rst:89]: Yes please "up to".  It is important that the 10min be qualified.

Comment (beckykd) [reply docs/faqs/max_execution_time.rst]: Done

**Comment by abbycross** [comment]
FYI @beckykd and @nonhermitian -- Jules had asked me to use "system execution time" -- ok to use that here too, or should I change it to "job execution time" on my end? 

Also, on the UI the devs were asked to be more specific about when the time refreshes, so now we are using `00:00:00 UTC on the first of the month` to be clearer.

File: docs/faqs/max_execution_time.rst [hunk 6]
```diff
@@ -86,4 +86,4 @@ Other limitations
 
 - Programs cannot exceed 750KB in size.
 - Inputs to jobs cannot exceed 64MB in size.
-- Open plan users get 10 minutes of job execution time per month (reset on the first day of the month).  Job execution time is the amount of time that the system is dedicated to processing your job. You can track your monthly usage on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
\ No newline at end of file
+- Open plan users can use up to 10 minutes of job execution time per month (reset on the first day of the month, UTC).  Job execution time is the amount of time that the system is dedicated to processing your job. You can track your monthly usage on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account, <https://quantum-computing.ibm.com/account>`__ pages.
```
Comment (abbycross) [inline docs/faqs/max_execution_time.rst]:
```suggestion
- Open plan users can use up to 10 minutes of job execution time per month (reset on the first day of the month, UTC).  Job execution time is the amount of time that the system is dedicated to processing your job. You can track your monthly usage on the `Platform dashboard, <https://quantum-computing.ibm.com/>`__ `Jobs, <https://quantum-computing.ibm.com/jobs>`__ and `Account <https://quantum-computing.ibm.com/account>`__ pages.
```

**Comment by beckykd** [comment]
> FYI @beckykd and @nonhermitian -- Jules had asked me to use "system execution time" [here](https://docs.quantum-computing.ibm.com/run/instances#open-plan) -- ok to use that here too, or should I change it to "job execution time" on my end?
> 
> Also, on the UI the devs were asked to be more specific about when the time refreshes, so now we are using `00:00:00 UTC on the first of the month` to be clearer.

I added the UTC bit, so it should be showing up here.   Jessie is the one who suggested 'job execution time'.  Maybe we should have Jules and Jessie duke it out.

**Review by kt474 — APPROVED** [review]

**Review by jyu00 — APPROVED** [review]
Thanks @beckykd!

**Comment by jyu00** [comment]
> Jessie is the one who suggested 'job execution time'. Maybe we should have Jules and Jessie duke it out.

lol. I don't have a strong preference either way. Let's just go with system execution time since Jules can definitely take me out 😂 (sorry @beckykd this means you'll have to change it again).

**Pull request merged**